### PR TITLE
Fix for broken field filterable

### PR DESCRIPTION
**Pull request by Grokzen**
As reported in #65 the field `filterable` is broken and not working with new versions of netbox. This fixes that problem by removing `filterable` and replace it with `filter_logic`.

### CODE REVIEW

File: startup_scripts/20_custom_fields.py [hunk 1]
```diff
@@ -36,8 +36,11 @@ def get_class_for_class_path(class_path):
         if cf_details.get('description', 0):
           custom_field.description = cf_details['description']
 
-        if cf_details.get('filterable', 0):
-          custom_field.is_filterables = cf_details['filterable']
+        # If no filter_logic is specified then it will default to 'Loose'
+        if cf_details.get('filter_logic', 0):
+          for choice_id, choice_text in CF_FILTER_CHOICES:
```
Comment (cimnine) [inline startup_scripts/20_custom_fields.py]: If I get [the python docs](https://docs.python.org/2/library/stdtypes.html#typesseq) right you should be able to do `CF_FILTER_CHOICES.index(choice_text.lower())`.

Comment (Grokzen) [reply startup_scripts/20_custom_fields.py]: No this is not possible as the values in the variable `CF_FILTER_CHOICES` is a tuple containing several tuple objects. The extract of the inner tuple object is some syntax sugar that python provides.

```
>>> from extras.constants import CF_FILTER_CHOICES
>>> CF_FILTER_CHOICES.index('Disabled')           
Traceback (most recent call last):                
  File "<console>", line 1, in <module>           
ValueError: tuple.index(x): x not in tuple        
>>> CF_FILTER_CHOICES.index('disabled')           
Traceback (most recent call last):                
  File "<console>", line 1, in <module>           
ValueError: tuple.index(x): x not in tuple        
```

Comment (cimnine) [reply startup_scripts/20_custom_fields.py]: Alright, thank you for trying it.

**Comment by cimnine** [comment]
As this is a breaking change, could you please increase the (minor) version number in the `VERSION` file and add a note to the [`README.md`](https://github.com/ninech/netbox-docker/#breaking-changes)?

**Comment by cimnine** [comment]
Please also rebase the branch onto the most recent master (or merge the changes into your branch).

**Comment by calmyournerves** [comment]
@Grokzen Sorry this didn't get any attention, could you rebase it again?

**Comment by Grokzen** [comment]
@calmyournerves Can you consider to use "Rebase and Merge" instead? The bad thing is that it won't make a Merge commit if you really require it. The good thing is that you can get the changes on-top of the latest changes without my input & the commit history is much cleaner :]

**Comment by thde** [comment]
Fixes #65

**Pull request merged**